.github/MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,69 @@
+# DevDocs maintainer guidelines
+
+In general, the same [guidelines](https://devdocs.magento.com/guides/v2.3/contributor-guide/maintainers.html) for [`magento/magento2`](https://github.com/magento/magento2) maintainers apply to devdocs maintainers. However, there are some additional guidelines specific to devdocs that will help you as a maintainer.
```

Comment:
Did we want to cap DevDocs throughout?

Reply:
Yeah, good call. That's what we did in the contributor guide, so we should keep it consistent.